website/docs/reference/resource-configs/pre-hook-post-hook.md

```diff
@@ -15,6 +15,8 @@ datatype: sql-statement | [sql-statement]
 
 <TabItem value="models">
 
+**Note**: In the following examples, the `|` symbol is meant to separate two different formatting options for SQL statements in pre-hooks and post-hooks. The first option (without brackets) will accept a single SQL statement as a string, while the second (with brackets) will accept multiple SQL statements as an array of strings.
```

Comment:
I love this suggestion! I think it's more important than a note and would move it to the top of the page as a paragraph. Then remove the other two notes. My text suggestion just makes the sentence active.

```suggestion
In the following examples,  we use the `|` symbol to separate two different formatting options for SQL statements in pre-hooks and post-hooks. The first option (without brackets) will accept a single SQL statement as a string, while the second (with brackets) will accept multiple SQL statements as an array of strings.
```